utils/api-examples/src/tohex.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiPromise, WsProvider } from '@polkadot/api'
+import { CuratorApplicationId } from '@joystream/types/content-working-group'
+import { BTreeSet } from '@polkadot/types'
+import { types } from '@joystream/types'
+
+async function main() {
```

Comment:
This script only needs to create some types and doesn't need to read state from chain, and therefore shouldn't require creating an api instance or connecting to a node. I this type of scenario, what is the proper way to create a type registry and register our types so we can construct them safely.

cc: @Lezek123

Reply:
I added a function to do this in https://github.com/Joystream/joystream/pull/1176/files#diff-9e9b5c1632859d4c7bee5d17dd9f7c5c, it's not well-tested yet, but I think it should work fine. I can move this to another PR if needed.

Reply:
Yes I spotted this also in pioneer react-api: 
https://github.com/Joystream/joystream/blob/iznik/pioneer/packages/react-api/src/typeRegistry.ts
https://github.com/Joystream/joystream/blob/iznik/pioneer/packages/react-api/src/Api.tsx#L179

So it looks "safe" and correct way.